rbd/watchers_mimic_test.go

```diff
@@ -0,0 +1,160 @@
+// +build !luminous,!mimic
+//
+// Ceph Nautilus is the first version that supports watchers through librbd.
+
+package rbd
+
+import (
+	"testing"
+
+	"github.com/ceph/go-ceph/rados"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestListWatchers(t *testing.T) {
+	conn, _ := rados.NewConn()
+	conn.ReadDefaultConfigFile()
+	conn.Connect()
+
+	poolname := GetUUID()
+	err := conn.MakePool(poolname)
+	assert.NoError(t, err)
+
+	ioctx, err := conn.OpenIOContext(poolname)
+	assert.NoError(t, err)
+
+	name := GetUUID()
+	image, err := Create(ioctx, name, 1<<22, 22)
```

Comment:
Deprecated Create function. Ideally, we switch over to CreateImage. Similarly, we want OpenImage instead of image.Open unless there's a good reason to use the deprecated APIs in this test.

Reply:
And *now* I see this comment. Yes, correcting this too :smile: